modules/nf-core/sentieon/bwamem/main.nf

```diff
@@ -47,6 +47,9 @@ process SENTIEON_BWAMEM {
         echo "Decoded and exported Sentieon test-license system environment variables"
     fi
 
+    LD_LIBRARY_PATH=/usr/local/lib/:\$LD_LIBRARY_PATH
+    export LD_LIBRARY_PATH
```

Comment:
Any risk into doing that in conda/docker.
or any risk of collision with local LD_LIBRARY_PATH?